[TS-53] Stop overlapping and responsive styling for event type toggle

William-Unosquare opened the pull request:
- Fixes overlapping of event type toggle & calendar navigation buttons.
- Improves responsiveness of event type toggle.

![chrome_2018-09-11_16-36-11](https://user-images.githubusercontent.com/39301552/45370804-e5f2da80-b5e0-11e8-94d7-6fc75203baf2.png)

![chrome_2018-09-11_16-36-19](https://user-images.githubusercontent.com/39301552/45370805-e8553480-b5e0-11e8-947b-60c5095f14fd.png)

![chrome_2018-09-11_16-36-28](https://user-images.githubusercontent.com/39301552/45370806-e9866180-b5e0-11e8-90c1-a6b46de98107.png)

William-Unosquare committed:
Stop overlapping and responsive styling for event type toggle

mcromie92 reviewed (approved):
looks good

ghost commented:
@William-Unosquare just looking at the screenshot. Any chance a bit of spacing could be added between the icon and the text 'Team Events'

girwin575 commented on web-app/src/pages/Dashboard/styled.js:
```diff
@@ -2,11 +2,31 @@ import styled from 'styled-components';
+    display: none;
+  }
+
+  span {
```
any chance we could not use !important?

girwin575 commented on web-app/src/pages/Dashboard/styled.js:
```diff
@@ -2,11 +2,31 @@ import styled from 'styled-components';
+    display: none;
+  }
+
+  span {
```
Small thing but try to give everything a class rather than put CSS on a element.

bgribz replied:
I don't think this is an issue when used with styled-components, maybe even preferable to keep code clean.

girwin575 replied:
It's not a stylistic reason, it's just for performance. A small thing but makes all the difference across an app. Appreciate it's CSS within JS but it should still stand. I've been too quite a few performance talks now and it's always brought up with CSS.

girwin575 replied:
Go with whatever is consistent to what already has been done but it's just good to note for future.

girwin575 replied:
@bgribz was beside me and mentioned he thinks a class is generated. I checked https://www.styled-components.com/docs/advanced and inspected a box and it does generate a class. So ignore what I said there in terms of styled components but the principle stands

William-Unosquare committed:
Clean CSS

William-Unosquare commented:
Cleaned up the CSS. I used class names because it came with the bonus that it solved the !important along with a cascade swap. Also changed btnText to btn-text because it's how the other classes in the file are named.

Also fixed the margin on the icon. Ooops.
